src/main/java/de/hpi/bpt/argos/core/ArgosImpl.java

```diff
@@ -139,4 +158,140 @@ protected void enableCORS(Service sparkService) {
 			response.type("application/json");
 		});
 	}
+
+	/**
+	 * This method loads all default event types from the disk.
+	 */
+	protected void loadDefaultEventTypes() {
+		try {
+
+			loadStatusUpdateEventType();
+
+			PropertyEditor propertyEditor = new PropertyEditorImpl();
+			String eventTypesDirectoryPath = propertyEditor.getProperty(Argos.getArgosBackendEventTypeDirectoryPropertyKey());
+			if (eventTypesDirectoryPath.length() == 0) {
+				throw new NullPointerException();
+			}
+
+			// since the path is delivered as URI, it is represented as a HTML string. Thus we need to replace %20 with file system spaces.
+			File eventTypesDirectory = new File(eventTypesDirectoryPath.replaceAll("%20", " "));
+
+			for (File eventTypeFile : eventTypesDirectory.listFiles()) {
+				if (!eventTypeFile.getName().endsWith(".json")) {
+					continue;
+				}
+
+				loadDefaultEventType(eventTypeFile);
+			}
+
+		} catch (NullPointerException e) {
+			logger.error("cannot find directory for default event types or not defined in properties");
+			logger.trace("Reason: ", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Reason: " 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)